packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -228,6 +248,34 @@ class CreateCommand extends FlutterCommand {
     }
   }
 
+  _ProjectType getProjectType(Directory projectDir) {
+    _ProjectType template;
+    _ProjectType detectedProjectType;
+    final bool metadataExists = projectDir.absolute.childFile('.metadata').existsSync();
+    if (argResults['template'] != null) {
+      template = _stringToProjectType(argResults['template']);
+    } else {
+      if (projectDir.existsSync() && projectDir.listSync().isNotEmpty) {
```

Comment:
Add a comment line for when this would happen?

Reply:
Done